.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: "Build and attach binary to release"
+on:
+  release:
+    types: ["created"]
+permissions:
+  contents: write  # to upload the binaries to the release
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: ["amd64", "arm64"]
```

Comment:
I'm assuming it's quick enough to compile for arm. IME people are going to request arm binaries if we publish any kind of binary so we may as well do this now.

Reply:
No x86_64?

Reply:
amd64 and x86_64 are the same thing.